src/extensions/ConflictFixer/index.ts

```diff
@@ -47,9 +50,69 @@ export default class ConflictFixer implements Partial<Extension> {
         this.counter++;
 
         if (this.counter > 10) { // TODO: make configurable
+
             const msg = `[ShortNight]: Too many times(${this.counter}) of try fix conflict.`;
 
             if (DEBUG) {
+
+                /**
+                 * Holds the filter data from the event which have generated the conflicts
+                 * */
+                const filterData = ((ud) => {
+
+                    const eventsData = ud.map(ud => {
+                        return this.conflicts.event_body.map(eb => {
+                            if (eb.self.drawInfo.date === ud.date) {
+                                return JSON.stringify(ud);
+                            }
+                        })
+                    });
+
+                    const flatenTheEvents = eventsData.flat(1);
+
+                    return flatenTheEvents.filter(Boolean) as string[];
+                    
+                })(this.ext.components.timeline[0].drawInfo.events);
+
+                let errorMsg = `[ShortNight]:`;
+
+                if (filterData.length === 1) {
```

Comment:
What's the logic behind this? Why does length being 1 mean description overlapping?